kratos/spatial_containers/geometrical_objects_bins.h

```diff
@@ -183,30 +183,33 @@ class GeometricalObjectsBins
         }
     }
 
-    /** This method takes a point and finds the nearest object to it.
+    /** This method takes a point and finds the nearest object to it in a given radius.
      * If there are more than one object in the same minimum distance only one is returned
+     * If there are no objects in that radius the 
```

Comment:
incomplete sentence

Reply:
Seems that I forgot to push my last changes.